piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,6 +77,20 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def verify_operator_installed(self, operator_name, namespace):
+        """
+        Check if operator is installed and returned true or false
+        :param operator_name: name of the operator.
+        :param channel_name: name of the channel
+        return: object of CSV
+        """  
+        csv = ClusterServiceVersion(self.kube_config_file)
+        subscription = self.sub_obj.get_subscription(operator_name, namespace)
+        csv_name = subscription.status.currentCSV
+        csv_obj = csv.get_cluster_service_version(csv_name, namespace)
```

Comment:
same line of thought, this csv_obj var would probably better be called csv_resp. I think csv_obj is better suited for the variable you declared in line 87 which is actually an object instance of the ClusterServiceVersion class

Reply:
Done